inst/css/sidebar.css

```diff
@@ -1,13 +1,28 @@
 /* teal sidebar css */
 
 
-.filter_hamburger, .wunder_bar_button {
+.filter_hamburger,
+.wunder_bar_button {
   font-size: 16px;
   padding: 8px !important;
   float: right !important;
   background-color: transparent !important;
 }
 
+.badge-count {
+  padding-left: 9px;
+  padding-right: 9px;
+  -webkit-border-radius: 9px;
+  -moz-border-radius: 9px;
+  border-radius: 9px;
+  font-size: 12px;
+  background: #ff0000;
+  color: #fff;
```

Comment:
In `teal.slice` we use `var(--bs-gray-200, #e9ecef)` to have a theme color with a hard coded fallback. Could you apply this here as well?